.github/workflows/test.yaml

```
@@ -29,6 +29,10 @@ on:
         type: string
         description: Label for selecting the self-hosted runners.
         default: "large"
+      self-hosted-runner-series:
```

```suggestion
      self-hosted-runner-image:
```
We are moving into custom image support for the self-hosted runners, so there may be different types of images supported. We also use the term `image` more than series in this context.

---

I see, updating the input name, thanks!